draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -61,6 +63,10 @@
 
   private DraggableListener listener;
 
+  private float x1;
+  private float x2;
+  private static final int MIN_DISTANCE = 10; //Influence to sliding
```

Comment:
Please, move this constant declaration with the other one. Line 50. 

You can rename it to MIN_SLIDING_DISTANCE or MIN_SLIDING_DISTANCE_ON_MINIMIZED_CLICK and remove the comment.